nvcf/_nvcf_creation.sh

```diff
@@ -0,0 +1,7 @@
+source .env
```

Comment:
Can we add a 1-line comment at the top of this script that describes what it is doing?